docs/components/InputGroup/Web.stories.tsx

```diff
@@ -85,3 +90,31 @@ export const DateRange = DateRangeTemplate.bind({});
 DateRange.args = {
   flowDirection: "horizontal",
 };
+
+const VerticalWithHorizontalRowsTemplate: ComponentStory<
```

Comment:
Appreciate the story for testing but I don't think we need this added, the existing Nested story covers a vertical group with horizontal groups inside. I did put a Button before and after the wrapping `InputGroup` though and all looked good 👌🏻